### DISCUSSION TITLE
Jmask Object in PatternObject Layer Group

### DISCUSSION THREAD

**Discussion by tjingboem** [kunstmusik/blue#638] [Ideas]
is there a reason for not having the Jmask Object available in the PatternObject Layer Group?

![Screenshot from 2022-01-12 10-27-22](https://user-images.githubusercontent.com/6670911/149105591-7ac142bc-35bc-477e-a704-1318a1837132.png)

**Comment by kunstmusik** [comment]
Not that I can remember? Shall I try to enable it and see what happens?

**Comment by tjingboem** [comment]
yes, why not? if it is that easy to add/activate...
I can do some testing

**Reply by kunstmusik** [reply]
I have enabled JMask to Pattern Layers in the roboto branch. Could you give it a try and let me know if this works for you?

**Comment by tjingboem** [comment]
yes i can confirm that it works, i have a sounding result.
Thanks!

**Comment by kunstmusik** [comment]
Completed
